css/style.css

```diff
@@ -0,0 +1,66 @@
+
+body {
+    background-color: #fdfdfd;
+    align-items: center;
+    justify-content: center;
+    display: block;
+}
```

Comment:
Please there should be an indentation of only two spaces before your CSS rules (you have 4 spaces). This is due to Microverse best coding practice and to avoid linter errors.

Reply:
- Good job being polite while giving your review 👏 and also stating the reasons for your changes 💯 
- Please, I'd suggest you try using emojis while giving your reviews, this will make you sound more friendly 😄 😄